sw/device/lib/dif/dif_plic.c

```diff
@@ -41,9 +43,9 @@ typedef struct plic_reg_info {
  * register offsets.
  */
 typedef struct plic_target_reg_offset {
-  ptrdiff_t ie;        /*<< Interrupt Enable register offset. */
-  ptrdiff_t cc;        /*<< Claim/complete register offset. */
-  ptrdiff_t threshold; /*<< Threshold register offset. */
+  ptrdiff_t irq_enable;
+  ptrdiff_t claim_cimplete;
```

Comment:
```suggestion
  ptrdiff_t claim_complete;
```

You'll need to follow this rename through everywhere else, and can you please not drop the comments.